index.html

```diff
@@ -5,22 +5,37 @@
         <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Here is my CSS for red and green on win or fail:

    .has-success {
        background-color: green;
    }

    .has-fail {
        background-color: red;
    }